### ISSUE TITLE
Workflow report time is incorrect

### ISSUE THREAD

**Issue by AnnaSyme** [galaxyproject/galaxy#19184]
**Describe the bug**

I think the Galaxy invocation time in the workflow report is incorrect. 

It is showing as:
Thu, 21 Nov 2024 14:10:53 GMT

Whereas the current time (in the workflow report) is showing as:
22 November 2024 at 01:12 UTC  (which is correct)

**Galaxy Version and/or server at which you observed the bug**

Galaxy Version: Galaxy Au, 24.1.4dev0

**Browser and Operating System**

Operating System: macOS
Browser: Chrome

**To Reproduce**
Create a workflow report and include the invocation time and the current time, then run the workflow and look at the report. 

**Expected behavior**
I expected the invocation time to be closer to the real time I saw the workflow run (which was immediately, so close to the current time). I also thought it would be in UTC not GMT.

**Comment by mvdbeek** [comment]
I'm confused about the different times you mention, could you add a screenshot or share the report ?

**Comment by AnnaSyme** [comment]
yep @mvdbeek - here is a screenshot of the workflow report editor
<img width="404" alt="Screenshot 2024-11-23 at 13 19 02" src="https://github.com/user-attachments/assets/7fce0e24-db30-4648-9a15-8a5c6dbefe4b">

**Comment by AnnaSyme** [comment]
and here is how the report looks:
<img width="452" alt="Screenshot 2024-11-23 at 13 18 50" src="https://github.com/user-attachments/assets/035bb940-ba36-4d23-9f5a-f1f0e1aa810f">

**Comment by AnnaSyme** [comment]
The UTC time is correct. Here is the workflow report page: https://usegalaxy.org.au/u/anna/p/invocation-report-e56b9c80fb6b598d
